packages/sui-studio/src/components/root/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import {Router, browserHistory} from 'react-router'
+import {Router} from '@s-ui/react-router'
 import routes from '../../routes'
 import '../../index.scss'
 
-export default () => <Router routes={routes} history={browserHistory} />
+export default () => <Router routes={routes} />
```

Comment:
`history` now is optional. It detects by default if we're on client or server in order to use the correct `history` for us.